app/models/conversion_host.rb

```diff
@@ -311,7 +311,7 @@ def ansible_playbook(playbook, extra_vars = {}, miq_task_id = nil, auth_type = n
 
     command = "ansible-playbook #{playbook} --inventory #{host}, --become --extra-vars=\"ansible_ssh_common_args='-o StrictHostKeyChecking=no'\""
 
-    auth = authentication_type(auth_type) || authentications.first
+    auth = authentication_type(auth_type) || authentications.first || find_credentials
```

Comment:
I still think it is weird that this is effectively:
```
auth = authentication_type(nil) || authentications.first || authentication_type('v2v') || resource.authentication_type('ssh_keypair') || resource.authentication_type('default')
```

We should compartmentalize this into the find_credentials method

Reply:
it could probably just be `auth = find_credentials` but I've opted to be paranoid for now.